diagnostic.md

```diff
@@ -32,8 +41,11 @@ y = 10
 
 After each line of code executes, what are the values of x and y?  Do the variables change?  Do their values?
 
-<!-- solution below -->
+After the code runs:
+x === 4
+y === 10
```

Comment:
`x` is assigned the value of `y` while `y` is still 3 so the final values are `x = 3` and `y = 10`.